Asteroids/Player.cpp

```diff
@@ -21,14 +21,11 @@ namespace Asteroids
 	Player::Player()
 		: m_angle(0.0f)
 		, m_rotation(250)
+		, m_currentShip(-1)
 	{
-		// Set model render vertices
+		// Set model render vertices (default ship)
 		//
-		m_points.push_back(Engine::Vector2(0.0f, 20.0f));
-		m_points.push_back(Engine::Vector2(12.0f, -10.0f));
-		m_points.push_back(Engine::Vector2(6.0f, -4.0f));
-		m_points.push_back(Engine::Vector2(-6.0f, -4.0f));
-		m_points.push_back(Engine::Vector2(-12.0f, -10.0f));
+		ChangeShip();
```

Comment:
This method name may be misleading.  I would suggest to name it something like RenderShip